neard/Cargo.toml

```diff
@@ -45,7 +45,7 @@ near-rosetta-rpc = { path = "../chain/rosetta-rpc", optional = true }
 near-telemetry = { path = "../chain/telemetry" }
 near-epoch-manager = { path = "../chain/epoch_manager" }
 near-performance-metrics = { path = "../utils/near-performance-metrics" }
-near-rust-allocator-proxy = { git = "https://github.com/near/near-memory-tracker.git", tag="0.2.4" }
+near-rust-allocator-proxy = { git = "https://github.com/near/near-memory-tracker.git", tag="v0.2.5" }
```

Comment:
can you publish near-memory-tracker to crates.io so that we can depend on a proper version?